pom.xml

```diff
@@ -126,7 +126,7 @@
         <mockwebserver.version>20121111</mockwebserver.version>
         <freemarker.version>2.3.22</freemarker.version>
         <commons-io.version>2.4</commons-io.version>
-        <hazelcast.version>3.0</hazelcast.version>
+        <hazelcast.version>3.5.4</hazelcast.version>
```

Comment:
scratch that, i see other projects mongo/qpid/etc used in tests for one subproject follow the smae pattern (and you just changed it) so that's a separate issue